browser_history/__init__.py

```diff
@@ -18,8 +18,30 @@ def get_history():
         try:
             browser_object = browser_class()
             browser_output_object = browser_object.fetch()
-            output_object.entries.extend(browser_output_object.get())
+            output_object.history_entries.extend(browser_output_object.get_history())
         except AssertionError:
             utils.logger.info("%s browser is not supported", browser_class.name)
-    output_object.entries.sort()
+    output_object.history_entries.sort()
+    return output_object
+
+def get_bookmarks():
+    """This method is used to obtain browser bookmarks of all available and supported
+    browsers for the system platform.
+
+    :return: Object of class :py:class:`browser_history.generic.Outputs` with the
+        data member bookmark_entries set to list(tuple(:py:class:`datetime.datetime`, str,str,str))
+
+    :rtype: :py:class:`browser_history.generic.Outputs`
+    """
+    output_object = generic.Outputs()
+    subclasses = generic.Browser.__subclasses__()
+    for browser_class in subclasses:
+        if('Firefox' in str(browser_class)):
```

Comment:
This comment is not for you @lucasace . It's for people who add bookmark support for the other browsers. Since, this is a temp fix, this would be easily missed by the others. So, just tagging it here. :smiley:

Reply:
Sure, cause even if I do chrome SQL, It will still cause a error for Safari

Reply:
Yes